job.go

```diff
@@ -121,12 +121,12 @@ func (j *Job) GetDetails() *JobResponse {
 }
 
 func (j *Job) GetBuild(ctx context.Context, id int64) (*Build, error) {
-	// use job embedded URL to properly handle jobs in folders
-	url, err := url.Parse(j.Raw.URL)
-	if err != nil {
-		return nil, err
-	}
-	jobURL := url.Path
+
+	// url.Path does not work when you have a customized domai
+	// Ex: Server : https://<domain>/jenkins/job/JOB1
+	// In the above example /jenkins is part fo the domain name
+	// url.Path returns /jenkins/job/JOB1 instead of expected /job/JOB1
```

Comment:
I think we should just stating the expected behavior in comment section, suggest to organize it as below:
```
Support customized server URL,
i.e. Server : https://<domain>/jenkins/job/JOB1
"https://<domain>/jenkins/" is the server URL,
we are expecting jobURL = "job/JOB1"
```

Reply:
PR updated